dictionary.json

```diff
@@ -17946,6 +18645,16 @@
     "examples": [],
     "fields": []
   },
+  {
+    "toaq": "te-",
+    "type": "prefix",
+    "english": "have been X'd by …; passive perfect",
+    "gloss": "PPF",
+    "short": "",
+    "keywords": [],
+    "notes": [],
+    "examples": []
+  },
   {
     "toaq": "te",
```

Comment:
Similarly to mu, I don't think word should stick around. The examples though could be moved to te-.

Reply:
Leaving this to be decided in #153.